packages/plugin-express/src/express.js

```diff
@@ -13,9 +13,11 @@ module.exports = {
   name: 'express',
   load: client => {
     const requestHandler = (req, res, next) => {
-      // Get a client to be scoped to this request. If sessions are enabled, use the
-      // resumeSession() call to get a session client, otherwise, clone the existing client.
-      const requestClient = client._config.autoTrackSessions ? client.resumeSession() : clone(client)
+      // clone the client to be scoped to this request. If sessions are enabled, start one
```

Comment:
Would we need to clone the client if we used the stored value? i.e.:

```js
if (client._config.autoTrackSessions) {
  client._clientContext.getStore().startSession()
  // or even:
  Bugsnag.startSession()
}
```

Reply:
What about other things like metadata addOnError and breadcrumbs? If we don't clone the client then wouldn't add these things added during the request then be present on all subsequent requests?

Reply:
I thought those things would only apply in the current context after the other changes

I guess you'd also need:

```js
req.bugsnag = client._clientContext.getStore()
```

Reply:
But it's the cloned client we put into the store and its that cloned client we retrieve whenever the context is accessed

Reply:
Ohh I see 🤦‍♀️ that makes sense!